web_programming/crawl_stackoverflow_job.py

```diff
@@ -0,0 +1,35 @@
+import requests #Navigating with Python
+from bs4 import BeautifulSoup #extract Stackoverflow pages
+
+#Get the Stackoverflow URL
+URL = f"https://stackoverflow.com/jobs?q=python&sort=i"
+
```

Comment:
Add another empty line before `get_last_page`

Reply:
Will psf/black add this automatically?

Reply:
I believe it will